jbpm-bpmn2/src/test/java/org/jbpm/bpmn2/handler/WorkItemHandlerExceptionHandlingTest.java

```diff
@@ -221,6 +222,7 @@ public void testErrornousHandlerWithStrategyMultipleRetry() throws Exception {
         ProcessInstance processInstance = ksession.startProcess("com.sample.boolean", params);
         assertEquals(ProcessInstance.STATE_COMPLETED, processInstance.getState());
         assertProcessVarValue(processInstance, "isChecked", "true");
-        assertEquals(3, workItemHandler.getCounter());
+        assertEquals(4, workItemHandler.getCounter());
```

Comment:
Please, clean up the system property `org.jbpm.exception.handlingStrategy.retry.count` already defined above. You could either use a `finally` block or remove it with the `After` junit method.